feature-analysis.ipynb

```diff
@@ -9,7 +9,11 @@
   },
   {
    "cell_type": "code",
+<<<<<<< HEAD
+   "execution_count": 1,
+=======
    "execution_count": 74,
+>>>>>>> main
```

Comment:
these merge markers will mess up your notebook

Reply:
Yeah, my bad. It should be fixed now